app/models/customer.py

```diff
@@ -0,0 +1,53 @@
+from flask import current_app
+from app import db
+
+class Customer(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.Integer)
+    phone_number = db.Column(db.String)
+    register_at = db.Column(db.DateTime, nullable = False)
+    videos_checked_out = db.Column(db.Integer, default=0)
+    videos = db.relationship("Rental", back_populates="customer")
+    
+    def make_json(self):
+        """
+        Takes in an instance of Customer 
+        returns id, name, registered_at, postal_code,
+        phone and videos_checked_out_count of that Customer 
+        instance in a JSON compatabile dictionary
+
+        """
+        return {
+            "id": self.id,
+            "name": self.name,
+            "registered_at": self.register_at,
+            "postal_code": str(self.postal_code),
+            "phone": self.phone_number,
+            "videos_checked_out_count": self.videos_checked_out
```

Comment:
Instead of `videos_checked_out_count` being a column we can use the rentals to count that.  

To do that it's helpful to add a relationship:
```rentals = db.relationship('Rental', back_populates='customer', lazy=True)```

For `to_dict`, once you have the relationship the `videos_checked_out_count` is the length of the `rentals`:
```videos_checked_out_count": len(rentals)```

This method of dynamically computing `videos_checkout_out_count` rather than storing it in an instance variable makes it so there isn’t information stored in multiple places that could potentially conflict.